### PR TITLE
lint fix

### LINKED ISSUE #121: Fix Lint
https://github.com/spo-iitk/ras-backend/actions/runs/5330445534

### PR DESCRIPTION
**Pull request by abhishekshree**
Closes #121

### CODE REVIEW

**Comment by abhishekshree** [comment]
confirm this: https://github.com/spo-iitk/ras-backend/pull/122/files#diff-6f683e3cb781da5edd8ba6511ff9611217dd3860d9f53aff92822f19373d966c

File: auth/user.login.go [hunk 1]
```diff
@@ -42,7 +42,18 @@ func loginHandler(c *gin.Context) {
 		return
 	}
 
-	go setLastLogin(loginReq.UserID)
+	err_chan := make(chan error, 1)
```
Comment (1-Harshit) [inline auth/user.login.go:45]: What?

Comment (1-Harshit) [reply auth/user.login.go:45]: No

**Review by 1-Harshit — DISMISSED** [review]
Just revert this and we're good to go

File: auth/user.login.go (hunk 1 above)
Comment (abhishekshree) [reply auth/user.login.go:45]: why

Comment (1-Harshit) [reply auth/user.login.go:45]: Unnecessarily delays the login flow, user isn't really at fault if last login is not updated. it's like telling someone something failed cuz updated at was not updated.

Comment (abhishekshree) [reply auth/user.login.go:45]: how delaying?

Comment (1-Harshit) [reply auth/user.login.go:45]: Db call? It's fast I know, still. Login hash comparison is slow as it is. Our slowest api call after stats is this one.
And what's the point of launching it asynchronously if the very next line is waiting for its completion 😞

File: auth/user.login.go [hunk 2]
```diff
@@ -42,6 +42,8 @@ func loginHandler(c *gin.Context) {
 		return
 	}
 
+	// @1-Harshit and Copilot ❤️
+	//nolint:all
```
Comment (1-Harshit) [inline auth/user.login.go:46]: ❤️
Btw are we not expecting a whitespace after `//`?

**Review by 1-Harshit — APPROVED** [review]

**Comment by 1-Harshit** [comment]
BC @abhishekshree

File: auth/user.login.go (hunk 2 above)
Comment (abhishekshree) [reply auth/user.login.go:46]: no

Comment (abhishekshree) [reply auth/user.login.go:46]: no

**Pull request merged**